build.ps1

```diff
@@ -1,5 +1,5 @@
 
-if ($env:APPVEYOR_REPO_BRANCH -eq "dev") {
+if ($env:APPVEYOR_REPO_BRANCH -eq "disabled") {
```

Comment:
@mathewc did you intend this change. I think Naren put this here to have the `dev` build be composed of `dev` language workers.

Reply:
Yes, intentional. Naren told me to do this. This block isn’t working correctly.